### PR TITLE
mds 3917 - Display Permit Package and Map documents in MineSpace

### PR DESCRIPTION
**Pull request by mmumby**
# Main
- Added Map files and Permit Package Documents to the Permit tab on MineSpace
- Renamed "final application package" label to be "Permit Package" to match the terminology on the NoW

# How to test

- Navigate to a mine record with a permit that was issued in Core, look at that mine on MineSpace.

# Notes
Screen Shot 2021-09-13 at 4 26 25 PM

### CODE REVIEW

File: services/minespace-web/src/components/dashboard/mine/permits/PermitsTable.js [hunk 1]
```diff
@@ -50,7 +50,20 @@ const columns = [
...
+      : [];
+  const finalAppPackageImported =
+    amendment?.imported_now_application_documents?.length > 0
+      ? amendment.imported_now_application_documents.filter((doc) => doc.is_final_package)
+      : [];
+  return finalAppPackageCore.concat(finalAppPackageImported);
+};
+
 const transformRowData = (permit, permitStatusOptions) => {
+  console.log(permit);
```
Comment (JustinMacaulay) [inline services/minespace-web/src/components/dashboard/mine/permits/PermitsTable.js]: Missed log statement

File: services/minespace-web/src/components/dashboard/mine/permits/PermitsTable.js [hunk 2]
```diff
@@ -97,13 +115,67 @@ export const PermitsTable = (props) => {
...
+        width: "250px",
+        render: (text) => <div title="Authorization End Date">{text}</div>,
+      },
       { title: "Description", dataIndex: "description", key: "description" },
       {
-        title: "Documents",
+        title: "Map Files",
+        dataIndex: "maps",
+        key: "maps",
+        render: (text) => {
```
Comment (JustinMacaulay) [inline services/minespace-web/src/components/dashboard/mine/permits/PermitsTable.js]: is Text the required variable name here? if not can it be a little more accurate to what the data is?

**Review by JustinMacaulay — COMMENTED** [review]
Just a few small comments

File: services/minespace-web/src/components/dashboard/mine/permits/PermitsTable.js (hunk 2 above)
Comment (mmumby) [reply services/minespace-web/src/components/dashboard/mine/permits/PermitsTable.js]: 'maps' is the variable name, but all of our table renders have `(text, record)` as their positional parameters - just following the naming conventions of ant design, 'text === maps' in the data set

File: services/minespace-web/src/components/dashboard/mine/permits/PermitsTable.js (hunk 1 above)
Comment (mmumby) [reply services/minespace-web/src/components/dashboard/mine/permits/PermitsTable.js]: removed

File: services/minespace-web/src/components/dashboard/mine/permits/PermitsTable.js (hunk 2 above)
Comment (JustinMacaulay) [reply services/minespace-web/src/components/dashboard/mine/permits/PermitsTable.js]: Fair, I get it.

**Review by JustinMacaulay — APPROVED** [review]
Looks good.

**Pull request merged**